manual/manual/anchored_book.hva

```diff
@@ -0,0 +1,30 @@
+%hevea book class with anchor links in headers
+\input{bookcommon.hva}
+\newcommand{\@book@attr}[1]{\@secid\envclass@attr{#1}}
+\newcommand{\@titlesecanchor}{\@open{a}{href="\#\@sec@id@attr"}}
+\@makesection
+  {\part}{-2}{part}
+  {\@opencell{class="center"}{}{}\@open{h1}{\@book@attr{part}}}%
```

Comment:
Not sure what `opencell` generates but `center` is not a very good class name. `part` would be more approriate.

Reply:
It would probably be a good idea to do some class name cleaning  in a later PR indeed.

Reply:
About that `opencell` I actually had a look at that it seems the toc is still using table layout. Nowadays we have much better layout tool in CSS so it might be worth at a certain point to turn that `toc` page into properly nested `<ol>` possibly simply seperated by heading for the the different parts.